src/test/resources/no/nav/familie/ef/sak/beregning_barnetilsyn_en_mnd.feature

```diff
@@ -0,0 +1,81 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning av barnetilsyn (en periode)
+
+  Scenario: Varierende utgifter og inntekter, satsperiode og antall barn for 2022
+
+    Gitt følgende data
+      | Rad | Periodeutgift | KontrantstøtteBeløp | TillegsstønadBeløp | AntallBarn | PeriodeDato | Testkommentar   |
```

Comment:
KontantstøtteBeløp / TilleggsstønadBeløp
Ikke så nøye nå, men vanlig cucumber "kodestandard" er at man ikke benytter camelcase men bare bruker mellomrom for kolonnenavn og andre variabler som det her.

Reply:
Takk for tips - vi fikser nå :)